tests/format/FunctionDefinitions/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -17,11 +17,11 @@ interface FunctionInterfaces {
 
   function manyParams(uint x1, uint x2, uint x3, uint x4, uint x5, uint x6, uint x7, uint x8, uint x9, uint x10);
 
-  function manyModifiers() modifier1 modifier2 modifier3 modifier4 modifier5 modifier6 modifier7 modifier8 modifier9 modifier10;
+  function manyModifiers() modifier1() modifier2() modifier3 modifier4 modifier5 modifier6 modifier7 modifier8 modifier9 modifier10;
```

Comment:
Shouldn't these parentheses be removed?

Reply:
This is the "input" part of the snapshot. The output, that didn't change (and therefore it's not shown here) doesn't have parentheses in these modifiers.